lib/executor/ramping_vus_test.go

```diff
@@ -36,6 +37,34 @@ func TestRampingVUsConfigValidation(t *testing.T) {
 	errs = c.Validate()
 	require.NotEmpty(t, errs)
 	assert.Contains(t, errs[0].Error(), "greater than 0")
+
+	t.Run("validate large VU shifts should fail", func(t *testing.T) {
```

Comment:
I would prefer if we would split this test case in three different smaller ones, with comments detailing what and why we're testing explicitly 🙏🏻